automation/models.py

```diff
@@ -75,3 +76,73 @@ def get_valid_statuses():
             "New",
             "Processed",
         }
+
+
+class IntakeModel(MetaBaseModel):
+    ticket_id: str = pydantic.Field(alias="Ticket ID")
+    intake_volunteer: List[str] = pydantic.Field(
+        alias="Intake Volunteer - This is you!", default_factory=list
+    )
+    delivery_volunteer: List[str] = pydantic.Field(
+        alias="Delivery Volunteer", default_factory=list
+    )
+    neighborhood: Optional[str] = pydantic.Field(alias="Neighborhood")
+    request_name: Optional[str] = pydantic.Field(
+        alias="Requestor First Name and Last Initial"
+    )
+    nearest_intersection: Optional[str] = pydantic.Field(
+        alias="Nearest Intersection"
+    )
+    language: List[str] = pydantic.Field(
+        alias="Language", default_factory=list
+    )
+    address: Optional[str] = pydantic.Field(
+        alias="Address (won't post in Slack)"
+    )
+    phone_number: Optional[str] = pydantic.Field(alias="Phone Number")
+    vulnerability: List[str] = pydantic.Field(
+        alias="Vulnerability", default_factory=list
+    )
+    household_size: Optional[int] = pydantic.Field(alias="Household Size")
+    delivery_notes: Optional[str] = pydantic.Field(
+        alias="Notes for Delivery Volunteer (won't post in Slack)"
+    )
+    food_options: List[str] = pydantic.Field(
+        alias="Food Options", default_factory=list
+    )
+    other_items: Optional[str] = pydantic.Field(alias="Other Items")
+    recordID: str = pydantic.Field(alias="record ID")
+    date_completed: Optional[date] = pydantic.Field(alias="Completion_Date")
+    can_meet_outside: Optional[bool] = pydantic.Field(alias="Can meet outside")
+
+    @staticmethod
+    def get_valid_statuses():
+        return {
+            "Seeking Volunteer",
+            "Assigned / In Progress",
+            "Complete",
+            "Not Bed-Stuy",
+            "Assistance No Longer Required",
+            "Cannot Reach / Out of Service",
+            "Bulk Delivery Scheduled",
+            "Bulk Delivery Confirmed",
+            "AC Needed",
+            "AC Delivered",
+            "Seeking Other Goods",
+        }
+
+
+class ItemsByHouseholdSizeModel(BaseModel):
```

Comment:
This uses `BaseModel` instead of `MetaBaseModel` because it doesn't have status or meta information and isn't polled for anything.